### DISCUSSION TITLE
I was able to get this and the follow/subs/viewers widget to work, but how can I add channel point redeems with TwitchLib?

### DISCUSSION THREAD

**Discussion by Aivriedear** [baffler/Transparent-Twitch-Chat-Overlay#183] [Q&A]
Sorry, I don't understand at all how TwitchLib works, so if someone could give me an idea of the steps I should take I would greatly appreciate it.

**Comment by baffler** [comment]
This upcoming update will have better support for TwitchLib and be easier to setup. It will be in the next version 0.9.5, which I will release soon. You'll just go to settings, click "Connections" and connect your twitch account there. then in settings under "Chat", you will see the "Redemptions" and you can toggle the checkbox there to enable it.
